### PR TITLE
chore(release): v0.15.0

### PR DESCRIPTION
**Pull request by jsdw**
Update changelog and versions. Add generate_changelgo and adapt release steps from subxt so that the process can be a little more mindless.

### CODE REVIEW

**Comment by jsdw** [comment]
Will hold off on this until #820 is in.

**Review by TarikGul — APPROVED** [review]
Giving an early thumbs up, #820 still needs but be added but it looks good to me.

**Comment by jsdw** [comment]
I rebased onto latest master after 820 was merged (for cleanliness) and updated the changelog. This hsould be good to go now.

**Review by Xanewok — APPROVED** [review]

**Pull request merged**
